reference/webhooks.v3.yml

```diff
@@ -135,12 +135,12 @@ x-stoplight:
 servers:
   - url: 'https://api.bigcommerce.com'
     variables:
-      $$.env.store_hash:
-        default: '{$$.env.store_hash}'
+      store_hash:

```

Comment:
on a couple different docs, i see these server variables. since we have them listed as path parameters, is it correct to keep them listed as server variables?

if we're keeping `variables: store_hash`, standardize across files. some have a `default` value of `{store_hash}` and others say `unknown`.

Reply:
Good call